code4romania/stam-acasa
# Avoid user enumeration by redirecting duplicated requests

When an account already exists the user is redirected to the registration success screen as there is no error, in order to avoid user enumeration

### What does it fix?

Closes #440 

Please mention the main changes this PR brings.
- redirects the user on **DuplicateUserName** error
- the email is not chcked anymore on the registration success screen
- *EmailConfirmation* is not configurable anymore in order to avoid different behaviors on the registration screen

### How has it been tested?

Manually tested it
- register abc@mailinator.com 
- confirm email
- confirm login works
- register again with abc@mailinator.com
- *this PR behavior*: the user is redirected to the registration confirmation screen as the first time
- *original behavior*: the user was present an error message informing him/her and account already exists
